Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/07-SyncUpDetailNavigation/SyncUpDetailNavigation.tutorial

```diff
@@ -18,7 +18,9 @@
         > Note: You may already have an App.swift in your project that holds the entry point of
         the application (i.e. `@main`). If so, you can name this file AppFeature.swift, or you can
         rename the entry point to Main.swift.
-        
+        >
+        > We often leave out suffixing reducers with names like "feature" or "reducer", but in this case we must avoid conflicting with SwiftUI's [App](https://developer.apple.com/documentation/swiftui/app) protocol, so `AppFeature` is used as the type name.
```

Comment:
Figured I'd toss this small blurb in here too, but it can be removed if you want.